todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/statistics/StatisticsViewModel.java

```diff
@@ -103,6 +109,17 @@ public void call() {
                 });
     }
 
+    @NonNull
+    private String getStatisticsString(int numberOfActiveTasks, int numberOfCompletedTasks) {
+        if (numberOfCompletedTasks == 0 && numberOfActiveTasks == 0) {
+            return mResourceProvider.getString(R.string.statistics_no_tasks);
+        } else {
+            return mResourceProvider.getString(R.string.statistics_active_tasks) + " "
```

Comment:
How about using `String.format` and make `statistics_active_tasks` have a `%d` parameters? It would make it more readable